src/SampleLinter/SampleLinter.csproj

```diff
@@ -1,82 +1,37 @@
-      <IncludeAssets></IncludeAssets>
-      <PrivateAssets></PrivateAssets>
-      <ExcludeAssets>Runtime</ExcludeAssets>
-    </PackageReference>
-    <PackageReference Include="DynamoVisualProgramming.DynamoServices" Version="2.18.1.5096">
-      <IncludeAssets></IncludeAssets>
-      <PrivateAssets></PrivateAssets>
-      <ExcludeAssets>Runtime</ExcludeAssets>
-    </PackageReference>
-    <PackageReference Include="DynamoVisualProgramming.WpfUILibrary" Version="2.18.1.5096">
-      <IncludeAssets></IncludeAssets>
-      <PrivateAssets></PrivateAssets>
-      <ExcludeAssets>Runtime</ExcludeAssets>
-    </PackageReference>
-    <PackageReference Include="DynamoVisualProgramming.ZeroTouchLibrary" Version="2.18.1.5096">
-      <IncludeAssets></IncludeAssets>
-      <PrivateAssets></PrivateAssets>
-      <ExcludeAssets>Runtime</ExcludeAssets>
-    </PackageReference>
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1">
-      <IncludeAssets></IncludeAssets>
-      <PrivateAssets></PrivateAssets>
-      <ExcludeAssets>Runtime</ExcludeAssets>
-    </PackageReference>
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="SampleLinter_ExtensionDefinition.xml">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Update="Properties\Resources.Designer.cs">
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-      <DependentUpon>Resources.resx</DependentUpon>
-    </Compile>
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Update="Properties\Resources.en-US.resx">
-      <Generator></Generator>
-    </EmbeddedResource>
-    <EmbeddedResource Update="Properties\Resources.resx">
-      <Generator>PublicResXFileCodeGenerator</Generator>
-      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
+      <PackageReference Include="DynamoVisualProgramming.DynamoServices" Version="3.0.0-beta7064" />
+      <PackageReference Include="DynamoVisualProgramming.WpfUILibrary" Version="3.0.0-beta7064" />
+      <PackageReference Include="DynamoVisualProgramming.ZeroTouchLibrary" Version="3.0.0-beta7064" />
   </ItemGroup>
   <ItemGroup>
     <None Update="LinterSettings.xml">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="SampleLinter_ExtensionDefinition.xml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
   </ItemGroup>
+  <Target Name="AfterBuildOps" AfterTargets="Build">
+    <Message Text="Should run after build" Importance="High" />
+    <ItemGroup>
+        <PackageDll Include="$(OutputPath)SampleLinter.dll" />
+        <PackageXml Include="$(OutputPath)SampleLinter_ExtensionDefinition.xml" />
+        <SettingsXml Include="$(OutputPath)LinterSettings.xml" />
+    </ItemGroup>
+    <Copy SourceFiles="@(PackageDll)" DestinationFolder="$(ProjectDir)..\..\dynamo_linter\Sample Linter\bin" />
```

Comment:
whats the structure of this output path - does it live in the package output folder? Is it a separate package from the main samples one, or is it in the same package?

Reply:
it is a separate dynamo_linter folder

Reply:
These packages are not exactly published, are they? Are they simply there for testing in Dynamo?

Reply:
I do not know if anyone has published these packages (manually ?). There is no automation for it.
From the looks of it, it is there just to provide an example of how to build a package/linter/extension.
I do not think we have an example for the publishing process